arch/arm/boot/dts/overlays/i2c-rtc-gpio-overlay.dts

```diff
@@ -204,6 +204,23 @@
 		};
 	};
 
+	fragment@13 {
+		target = <&i2c_gpio>;
+		__dormant__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+
+			rv1805: abx80x@69 {
```

Comment:
One open question here, as I wasn't able to find sufficient Device Tree documentation: should the 'abx80x' on this line be 'rv1805'? I'm not sure what this label refers to.

Reply:
The part after the colon and before the left brace - `abx80x@69` - is the name of the node; it has to be unique amongst the subnodes/children of the i2c_gpio node. The bit before the colon is the label, used when something else (another fragment or a parameter) refers to a node; labels have to be unique for the whole dtb or overlay, although normally multiple overlays can define the same label without clashing (labels aren't normally preserved when the overlay is applied).

A node name can be anything you like - they have no significance - but there are conventions. The part after the `@` (if there is one) is referred to as the address, and should match the value of the `reg` property. The remainder should be something meaningful - in this case I would use `rv1805`.

Reply:
Plus, the automated checker is happier if the "name" portion matches the name of the parameter that enables the fragment, so `rv1805` is much preferred.

Reply:
All done, thanks.